api/server/docker.go

```diff
@@ -169,79 +215,132 @@ func (d *driver) mountpath(name string) string {
 	return path.Join(volume.MountBase, name)
 }
 
+func (d *driver) getConn() (*grpc.ClientConn, error) {
+	if d.conn == nil {
+		var err error
+		d.conn, err = grpcserver.Connect(
+			d.sdkUds,
+			[]grpc.DialOption{grpc.WithInsecure()})
+		if err != nil {
+			return nil, fmt.Errorf("Failed to connect to gRPC handler: %v", err)
+		}
+	}
+	return d.conn, nil
+}
+
 func (d *driver) create(w http.ResponseWriter, r *http.Request) {
 	method := "create"
+	ctx := r.Context()
+
+	// decode request
 	request, err := d.decode(method, w, r)
 	if err != nil {
 		return
 	}
 
+	// attach token in context metadata
+	d.attachToken(ctx, request)
```

Comment:
this function mutates `ctx`. Instead it should return ctx:

```go
ctx, err = d.attachToken(ctx, request)
```

Which is aligned with other context calls.